.github/workflows/ci.yaml

```diff
@@ -0,0 +1,289 @@
+name: CI Tasks for Ory Hydra
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+
+
+jobs:
+  oidc-conformity:
+    name: Run OIDC conformity tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          # We must fetch at least the immediate parents so that if this is
+          # a pull request then we can checkout the head.
+          fetch-depth: 2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.16.1'
+      - name: Start service
+        run: ./test/conformance/start.sh
+      - name: Run tests
+        run: ./test/conformance/test.sh -v -short -parallel 16
+
+  test:
+    name: Run tests and lints
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ^1.17
+      - run: go list -json > go.list
+      - name: Run nancy
+        uses: sonatype-nexus-community/nancy-github-action@v1.0.2
+      - name: Run prettier
+        uses: ory/ci/prettier@master
+      - name: Run prettier (docs)
+        uses: ory/ci/prettier@master
+        with:
+          dir: docs
+      - name: Run golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        env:
+          GOGC: 100
+        with:
+          args: --timeout 10m0s --issues-exit-code=0
+          skip-go-installation: true
+          skip-pkg-cache: true
+      - run: |
+          make .bin/go-acc
```

Comment:
Could you give these/all run steps a name? I was looking at the CI runner and was wondering where the test were, until I realived that `make .bin/go-acc` includes also running the tests

Reply:
Done!